supplier_app/tests/test_view.py

```diff
@@ -212,8 +215,70 @@ def test_address_bankaccount_should_be_related_with_taxpayer(self, msg_mocked):
             bankaccount.taxpayer
         )
 
+    def test_logged_in_supplier_can_create_taxpayer(self):
+
+        response = self.client.get(
+            reverse('taxpayer-create'),
+        )
+
+        self.assertIn(
+            'supplier_app/taxpayer-creation.html',
+            response.template_name,
+        )
+        self.assertEqual(
+            response.status_code,
+            HTTPStatus.OK
+        )
+        self.assertTrue(
+            self.user_with_eb_social.groups.filter(name='supplier').exists()
+        )
+
+    def test_not_logged_in_supplier_cant_create_taxpayer(self):
+        client = Client()
+        supplier_user = UserFactory()
+        supplier_user.groups.add(self.supplier_group)
+
+        response = client.get(
+            reverse('taxpayer-create'),
+            follow=True
+        )
+        self.assertIn(
```

Comment:
Maybe you can add something like the previous test, I don't remember if assertNotIn exists, but something like

`self.assertNotIn(
            'supplier_app/taxpayer-creation.html',
            response.template_name,
        )`